js/web-analytics-opt-out.js

```diff
@@ -0,0 +1,3 @@
+document.addEventListener('DOMContentLoaded', function (event) {
+    confirm("We use cookies and other tracking technologies to improve your browsing experience on our website, to analyze our website traffic, and to understand where our visitors are coming from. Read more about how we use cookies in our Privacy Policy.");
+});
```

Comment:
this is nice, it tells users first thing of opt out option. I just wonder if you give them the option to leave or stay.